### DISCUSSION TITLE
How to get the `~/` content in frontmatter and escape it properly?

### DISCUSSION THREAD

**Discussion by kecrily** [ElMassimo/iles#94] [Q&A] [answered]
I defined a `banner` field in the frontmatter of the markdown file to hold the banner image.

They look like this

```md
---
title: Hello World
banner: ~/images/hello-world.webp
---
Hello
```

Then I get this field in layout like this

```vue
<template>
  <img :src=$frontmatter.banner />
</template>
```

Rendered like this

```html
<img src="~/images/hello-world.webp">
```

How can I get it to render as expected?

**Comment by ElMassimo** [comment, marked as answer]
Please read this [similar discussion](https://github.com/ElMassimo/iles/discussions/82#discussioncomment-2092564) which provides several options to tackle this use case.
